configs/fluxcd.json

```
@@ -1,14 +1,18 @@
 {
   "index_name": "fluxcd",
   "start_urls": [
-    "https://fluxcd.io/"
+    {
+      "url": "https://fluxcd.io/",
+      "tags": ["current"]
+    },
+    {
+      "url": "https://fluxcd.io/legacy",
+      "tags": ["legacy"]
+    }
```

```suggestion
    {
      "url": "https://fluxcd.io/legacy",
      "tags": ["legacy"]
    },
    {
      "url": "https://fluxcd.io/",
      "tags": ["current"]
    }
```

It needs to be the other way around since `legacy` is a subdomain of the root one